<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_room</class>
 <widget class="QWidget" name="user_room">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="resultWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>561</width>
     <height>161</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="resultsShowLabel">
      <property name="text">
       <string>Нет результатов, нажмите на &quot;Пройти тестирование&quot;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>17</y>
     <width>561</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Добро пожаловать в кабинет испытуемого.</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="showresButton">
        <property name="text">
         <string>Посмотреть результаты</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="testingButton">
        <property name="text">
         <string>Пройти тестирование</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="changedataButton">
        <property name="text">
         <string>Изменить данные</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="changedata" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>561</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="name_label">
        <property name="text">
         <string>Имя</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="login_label">
        <property name="text">
         <string>Логин</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="password_label">
        <property name="text">
         <string>Пароль</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="sql_name_label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sql_login_label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordChangeLineedit"/>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Изменить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
